EKS-POD-IDENTITY-README.md

```diff
@@ -0,0 +1,160 @@
+## Domino Advanced Credential Propagation based on EKS Pod Identity
+
+[EKS Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html) provide the ability to manage 
+credentials for your applications, similar to the way that Amazon EC2 instance profiles provide credentials to 
+Amazon EC2 instances. Instead of creating and distributing your AWS credentials to the containers or using the Amazon 
+EC2 instance's role, you associate an IAM role with a Kubernetes service account and configure your Pods to use 
+the service account.
+
+## Article Topics
+ - Security 
+ - Identity And Access Management (IAM) 
+ - Role Based Access Control (RBAC)
+ - Open Authorization (OAuth), Web Identity and Attribute Based Access Control (ABAC) 
+
+## High Level Overiew
+
+Domino Data Lab’s platform is built on top of the Kubernetes framework. Domino Nexus is a comprehensive platform that 
+allows you to execute Data Science and Machine Learning workloads across any compute cluster - whether in the cloud, 
+a specific region, or on-premises. By unifying data science silos across the enterprise, Domino Nexus provides a 
+centralized hub for building, deploying, and monitoring models. 
+
+In a previous [article] (http://ADD_LINK_HERE) we saw how IAM Role for Service Accounts (IRSA) can be utilized within 
+Domino Workloads for assuming IAM Roles. There are situations where IRSA can be operationally challenging to use.
+AWS provides another mechanism to assume IAM roles within Kubernetes workloads known as EKS Pod Identity to address
+these challenges.
+
+In this article we will explore what these operational challenges are and how EKS Pod Identity can be used within 
+Domino Workloads. We will look at specific situations where this approach is more suitable than the IRSA approach.
+
+While this document is meant to guide you on configuring AWS IAM Role propagation to Domino Workloads based on 
+EKS Pod Identity, please speak with your Domino Professional 
+Services contact for additional information, assistance and support.
+
+
+### Operational Challenges of IRSA
+
+In general if you treat the EKS cluster hosting Domino as a "pet" and not "cattle", IRSA is a better choice. It offers
```

Comment:
Not sure i follow the pet/cattle terms here

Reply:
I reworded it. Let me know what you think of the changes.